spec/main-process/atom-application.test.js

```diff
@@ -368,6 +370,23 @@ describe('AtomApplication', function () {
     })
   })
 
+  describe('before quitting', function () {
+    it('waits until all the windows have saved their state and then quits', async function () {
```

Comment:
@steelbrain These are integration tests, and are being executed outside of Atom's test runner harness. They are using a more modern release of Jasmine ✨